db/kubernetes/client.go

```diff
@@ -22,9 +22,10 @@ import (
 	"os"
 	"strings"
 
-	"github.com/openebs/node-disk-manager/blockdevice"
+	. "github.com/openebs/node-disk-manager/blockdevice"
```

Comment:
can we continue to use the scope of blockdevice - It helps with the readability of the code.

Reply:
do you mean, no need to use the dot imports?

Reply:
Done.